guides/Getting Started/faq.md

```diff
@@ -51,7 +51,11 @@ const { token } = require('./config.json');
 
 For a tutorial
 
-1. Set `process.env.DISCORD_TOKEN` equal to your bot's token, you can do this using [dotenv](https://www.npmjs.com/package/dotenv). Run `npm install dotenv --save`
+Set `process.env.DISCORD_TOKEN` equal to your bot's token, you can do this by either
+
+* using the console to set the environment variable every time you run the bot, by doing `set DISCORD_TOKEN=token` 
```

Comment:
This line screws up the ordered list. This might be possible to fix if indented. I am not sure.

Reply:
I'm very sure that an indentation of 4 spaces in each line between the points fix this.